sys/net/gnrc/netif/gnrc_netif.c

```diff
@@ -1304,7 +1304,11 @@ static void _test_options(gnrc_netif_t *netif)
                    (IEEE802154_LONG_ADDRESS_LEN == netif->l2addr_len));
 #if IS_USED(MODULE_GNRC_NETIF_IPV6)
 #if IS_USED(MODULE_GNRC_NETIF_6LO)
+#if CONFIG_GNRC_NETIF_NONSTANDARD_6LO_MTU
```

Comment:
```suggestion
#if IS_ACTIVE(CONFIG_GNRC_NETIF_NONSTANDARD_6LO_MTU)
```

This way it also evaluates to 0 if not defined

Reply:
Solved at 1b2f020b5260ac833c33e37c04d1c116b62bdcc4